woocommerce/woocommerce
# Update pnpm-lock.yaml lockfile

### All Submissions:






### Changes proposed in this Pull Request:



This PR updates the `pnpm-lock.yaml` lockfile so that it doesn't get modified when running `pnpm install`. It is likely that somebody changes a dependency in `package.json` directly and pnpm wasn't aware of it.

### How to test the changes in this Pull Request:

1. `pnpm install` - make sure `pnpm-lock.yaml` isn't updated
2. `pnpm run build` - make sure build works

### Other information:




### FOR PR REVIEWER ONLY:
